### ISSUE TITLE
Add Source Value

### ISSUE THREAD

**Issue by PRijnbeek** [anthonysena/DrugUtilization#9]
We now have source code, name, but we need to add source_ value for those that do not have the source vocabs in yet. This would also allow us to do some quality control.

**Comment by anthonysena** [comment]
I'm checking with members of our team to see if this poses any restrictions - I seem to recall that there are some DBs that do not want you to share the source values this way. That said, this can be added to the package and make it optional to turn off. We can then raise a warning when it is turned on for audit purposes.

**Comment by PRijnbeek** [comment]
This would not be a problem for us as long as we can check what we share, which we can in the csv files.

**Comment by anthonysena** [comment]
Fair enough. I've confirmed that there are some data providers that will not allow us to report the source_values. I'll work to make this an option for running the overview - we can provide a way to override this option so we can always export the source values. 

The other thing we should consider: a small cell count. Some data providers restrict what can be reported (i.e n < 10) so we could enforce this during the export. Let me know if you agree with this approach.

**Comment by PRijnbeek** [comment]
yes I agree with a switch for this (for all output on DUS).

**Comment by cgreich** [comment]
Ditto.
